civis/futures.py

```diff
@@ -91,29 +91,27 @@ class CivisFuture(PollableResult):
     ----------
     job_id : int
         First element of the tuple given to `poller_args`
-    run_id : int
-        Second element of the tuple given to `poller_args`, if any
+    run_id : int or None
+        Second element of the tuple given to `poller_args`
+        (`None` if the poller function does not require a run ID)
 
     Examples
     --------
-    This example is provided as a function at :func:`~civis.io.query_civis`.
```

Comment:
I think this is still true, isn't it? The mechanics of how we're polling for output in that function may be different, but I think an important purpose of this comment is to prevent people from copy-pasting the below code when they want to run a query.

Reply:
I took it away as I found it somewhat confusing, though I don't have a strong preference. Comment added back at 1d53193